### PR TITLE
Use Open Clover for code coverage analysis

### PR DESCRIPTION
**Pull request by prodriguezdefino**
jacoco was interfering with surefire and failsafe making the IT tests fail when instrumenting through Jacoco. Open Clover works just fine and is compatible with codecov.

### CODE REVIEW

File: flink-connector-bigquery-examples/src/main/java/org/apache/flink/examples/gcp/bigquery/BigQueryExample.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            String projectName,
+            String datasetName,
+            String tableName,
+            String recordPropertyToAggregate,
+            String rowRestriction,
+            Integer limit)
+            throws Exception {
+
+        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.enableCheckpointing(60000L);
```
Comment (jayadeep-jayaraman) [inline flink-connector-bigquery-examples/src/main/java/org/apache/flink/examples/gcp/bigquery/BigQueryExample.java:94]: Can we get this as an user provided configuration

Comment (jayehwhyehentee) [reply flink-connector-bigquery-examples/src/main/java/org/apache/flink/examples/gcp/bigquery/BigQueryExample.java:94]: Addressed the same concern in previous PR (#47).

**Review by jayadeep-jayaraman — APPROVED** [review]

**Pull request merged**